home/migrations/0002_test_data.py

```diff
@@ -0,0 +1,36 @@
+from django.db import migrations, transaction
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ('home', '0001_initial'),
+    ]
+
+    def generate_data(apps, schema_editor):
+        from home.models import Profile
+        user_test_data = [
+            ('Rebecca', 'RebeccaRebecca', 'Rebecca@gmail.com', 'F'),
+            ('Danit', 'DanitDanit', 'Danit@gmail.com', 'F'),
+            ('Aviv', 'AvivAviv', 'Aviv@gmail.com', 'M'),
+            ('Ido', 'IdoIdo', 'Ido@gmail.com', None),
+        ]
+
+        with transaction.atomic():
+            # Test: Create a User and Profile from a User via username.
+            for username, password, email, gender in user_test_data:
+                user = User(username=username, password=password, email=email).save()
```

Comment:
https://docs.djangoproject.com/en/3.1/topics/auth/default/
```
>>> from django.contrib.auth.models import User
>>> user = User.objects.create_user('john', 'lennon@thebeatles.com', 'johnpassword')
```
I believe that using the create_user function enccrypts the passord properly, while just using save() does not.

Reply:
Thank you. 
I've made all the changes you suggested.